src/driver/kvm.rs

```diff
@@ -320,7 +325,13 @@ mod tests {
             ))
         });
 
-        let result = Kvm::new("some_vm", kvmi_mock);
+        let result = Kvm::new(
+            "some_vm",
+            kvmi_mock,
+            Some(DriverInitParam::KVMiSocket(
+                "/tmp/introspector".parse().unwrap(),
+            )),
```

Comment:
```suggestion
            Some(DriverInitParam::KVMiSocket("/tmp/introspector")),
```

Reply:
![Capture d’écran de 2020-08-10 21-00-40](https://user-images.githubusercontent.com/964610/89820006-94e11d80-db4c-11ea-838a-e374134e982f.png)
it's weird, because the code compiles, but my IDE suggests it's an error

Reply:
It do be like that sometimes with the intellij rust plugin ;)